packages/cursorless-vscode-e2e/src/suite/fixtures/scopes/javascript/line.scope

```diff
@@ -0,0 +1,117 @@
+
+aaa
+
+---
+
+[#1 Content]
+ 
+[]
```

Comment:
huh this becomes slightly confusing because otherwise when we see a `[` it's above the first line it's annotating 🤔

Reply:
I made that change now when I think it turned out quite nice. You're are correct that using the same character which different alignment is confusing